test/e2e_federation/service.go

```diff
@@ -53,6 +53,7 @@ var _ = framework.KubeDescribe("Federated Services [Feature:Federation]", func()
 	var _ = Describe("Without Clusters [NoCluster]", func() {
 		BeforeEach(func() {
 			fedframework.SkipUnlessFederated(f.ClientSet)
+			clusters = fedframework.ClusterSlice{}
```

Comment:
right, I forgot to make this modification that you already spot in a previous comment.